Google.PowerShell/Logging/GcLogCmdlets.cs

```diff
@@ -90,6 +90,18 @@ protected string PrefixProject(string logName, string project)
         }
 
         /// <summary>
+        /// Prefix projects/{project id}/sinks to sinkName if not present.
+        /// </summary>
+        protected string PrefixProjectToSinkName(string sinkName, string project)
+        {
+            if (!string.IsNullOrWhiteSpace(sinkName) && !sinkName.StartsWith($"projects/{project}/sinks"))
+            {
+                sinkName = $"projects/{project}/sinks/{sinkName}";
```

Comment:
I don't think their is any performance difference.  It just makes using the APIs easier.